harness/determined/cli/_util.py

```diff
@@ -99,6 +100,22 @@ def setup_session(args: argparse.Namespace) -> api.Session:
     )
 
 
+def session(
+    fn: Callable[[argparse.Namespace, api.Session], None]
+) -> Callable[[argparse.Namespace, api.Session], None]:
```

Comment:
```suggestion
) -> Callable[argparse.Namespace, None]:
```

cause the wrapped function does not take 2nd `api.Session` param anymore?

Reply:
done